plugins/modules/ec2_vpc_vpn.py

```diff
@@ -229,199 +243,297 @@
+        }]
+  contains:
+    accepted_route_count:
+      type: int
+      returned: O(state=present)
+      description: The number of accepted routes.
+    last_status_change:
+      type: str
+      returned: O(state=present)
+      description: The date and time of the last change in status.
+    outside_ip_address:
+      type: str
+      returned: O(state=present)
+      description:
+        - The Internet-routable IP address of the virtual private gateway's outside interface.
+    status:
+      type: str
+      returned: O(state=present)
+      description: The status of the VPN tunnel.
+    status_message:
+      type: str
+      returned: O(state=present)
+      description: If an error occurs, a description of the error.
+    certificate_arn:
+      description: The Amazon Resource Name of the virtual private gateway tunnel endpoint certificate.
+      returned: when a private certificate is used for authentication
+      type: str
+      sample: "arn:aws:acm:us-east-1:123456789012:certificate/c544d8ce-20b8-4fff-98b0-example"
 vpn_connection_id:
   description: The identifier for the VPN connection.
   type: str
-  returned: I(state=present)
+  returned: O(state=present)
   sample:
-    vpn_connection_id: vpn-781e0e19
+    vpn_connection_id: "vpn-781e0e19"
 """
 
 try:
-    from botocore.exceptions import BotoCoreError
-    from botocore.exceptions import ClientError
     from botocore.exceptions import WaiterError
 except ImportError:
     pass  # Handled by AnsibleAWSModule
 
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import NoReturn
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
 from ansible.module_utils._text import to_text
 from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
 
-from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
-from ansible_collections.amazon.aws.plugins.module_utils.tagging import ansible_dict_to_boto3_tag_list
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AnsibleEC2Error
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import create_vpn_connection
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import create_vpn_connection_route
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import delete_vpn_connection
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import delete_vpn_connection_route
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import describe_vpn_connections
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import ensure_ec2_tags
 from ansible_collections.amazon.aws.plugins.module_utils.tagging import boto3_tag_list_to_ansible_dict
 from ansible_collections.amazon.aws.plugins.module_utils.tagging import compare_aws_tags
 
 from ansible_collections.community.aws.plugins.module_utils.modules import AnsibleCommunityAWSModule as AnsibleAWSModule
```

Comment:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
```

Reply:
This has to be updated once the module is migrated to `amazon.aws`. Until it lives there this should not be changed

Reply:
@abikouo Is there a reason for waiting until migration?

Reply:
@GomathiselviS yes. Each module should use the corresponding class from its collection.
This is needed for the AWS agent which is set into AWS requests. The agent contains the name of the collection from which the module is executed.